pkg/k8s/synced/crd.go

```diff
@@ -231,25 +226,35 @@ type crdState struct {
 	m map[string]bool
 }
 
-func newCRDState(m map[string]bool) crdState {
+func newCRDState() crdState {
+	m := make(map[string]bool, len(crdResourceNames))
+	for _, name := range crdResourceNames {
+		m[name] = false
+	}
 	return crdState{
 		m: m,
 	}
 }
 
 var (
-	cnpCRD  = crdResourceName(v2.CNPName)
-	ccnpCRD = crdResourceName(v2.CCNPName)
-	cepCRD  = crdResourceName(v2.CEPName)
-	cnCRD   = crdResourceName(v2.CNName)
-	cidCRD  = crdResourceName(v2.CIDName)
-	clrpCRD = crdResourceName(v2.CLRPName)
+	crdResourceNames = []string{
+		crdResourceName(v2.CNPName),
+		crdResourceName(v2.CCNPName),
+		crdResourceName(v2.CEPName),
+		crdResourceName(v2.CNName),
+		crdResourceName(v2.CIDName),
+		crdResourceName(v2.CLRPName),
+	}
 )
 
 func crdResourceName(crd string) string {
 	return "crd:" + crd
 }
 
+func GetCRDResourceNames() []string {
+	return crdResourceNames
+}
+
```

Comment:
Nit: Let's move this up in the file since it's exported. Generally, exported functions should come first, and then unexported / private APIs can follow.

Reply:
Done.